### DISCUSSION TITLE
`cache_size_limit reached` when evaluating the accuracy

### DISCUSSION THREAD

**Discussion by WillLester** [open-compass/opencompass#2058] [Q&A]
Hello. I got `torch._dynamo.exc.RecompileLimitExceeded: cache_size_limit reached` when testing the accuracy. How can I solve this issue?

My script is
```
import torch
from mmengine.config import read_base
from opencompass.models import HuggingFacewithChatTemplate

with read_base():
    from opencompass.configs.datasets.ARC_c.ARC_c_gen import ARC_c_datasets
    from opencompass.configs.datasets.ARC_e.ARC_e_gen import ARC_e_datasets
    from opencompass.configs.datasets.mmlu.mmlu_gen import mmlu_datasets
    from opencompass.configs.datasets.gsm8k.gsm8k_gen import gsm8k_datasets

models = [
    dict(
        type=HuggingFacewithChatTemplate,
        abbr='gemma-3-1b-it-default',
        path='google/gemma-3-1b-it',
        max_out_len=1024,
        batch_size=1,
        run_cfg=dict(num_gpus=1),
    )
]

datasets = sum((v for k, v in locals().items() if k.endswith('_datasets')), [])
```
Thanks!
